scripts/variantstore/wdl/extract/hail_create_vat_inputs.py

```diff
@@ -124,37 +124,52 @@ def write_sites_only_vcf(ac_an_af_split, sites_only_vcf_path):
 
     ht = ac_an_af_rows.rows()
     ht = ht.filter(ht.alleles[1] != "*") # remove spanning deletions
+    ht = ht.checkpoint("gs://fc-eada2674-7c2b-42a6-8db3-0246872596dc/checkpoint")
```

Comment:
not sure we still need this (or that we don't)

Reply:
yeah, I will leave that question for Patrick!